docs/user/file-size.md

```diff
@@ -105,3 +105,7 @@ becomes useless because there is only one source for all pages.
 Cropping is an ineffective method for reducing the file size because it only
 adjusts the viewboxes and not the external parts of the source image. Therefore,
 the content that is no longer visible will still be present in the PDF.
+
+## Going Further
+
+The presentation [Putting a Squeeze on Your PDF](https://youtube.com/watch?v=tgOABUhVwFs) has other suggestions. One takeaway is that most of the significant size optimizations usually come from image and font modification. However, font optimization, such as replacing, merging, and subsetting, is not within the functionality of pypdf.
```

Comment:
I do not want to completely deny this as a future optimization, although this probably is of lower priority.

```suggestion
The presentation [Putting a Squeeze on Your PDF](https://youtube.com/watch?v=tgOABUhVwFs) has other suggestions. One takeaway is that most of the significant size optimizations usually come from image and font modification. However, font optimization, such as replacing, merging, and subsetting, is not within the functionality of pypdf at the moment.
```